doc/main.html

```diff
@@ -148,50 +150,103 @@ <h4>Overall structure</h4>
           <caption>Ordered sequence of <code>section</code> elements comprising the <code>body</code> element.</caption>
           <thead>
             <tr>
-              <th>Cardinality</th>
-              <th>Section</th>
+              <th>Section Number</th>
+              <th>Section Name</th>
+              <th>Mandatory, Conditional, or Optional</th>
             </tr>
           </thead>
           <tbody>
             <tr>
-              <td>0..1</td>
-              <td><a>introduction section</a></td>
+              <td>Unnumbered</td>
+              <td><a>Table of contents</a></td>
+              <td>Mandatory (generated automatically)</td>
+            </tr>
+            <tr>
+              <td>Unnumbered</td>
```

Comment:
Why is the numbering important if it is generated automatically?

Reply:
This is for informative purposely only, really meant to show user in the order listed. I have added "(informative)" to the header.